source/adl-standalone/tests/README.rst

```diff
@@ -0,0 +1,133 @@
+=====================
+Atlas Data Lake Tests
+=====================
+
+.. contents::
+
+----
+
+Introduction
+============
+
+The YAML and JSON files in this directory tree are platform-independent tests
+that drivers can use to prove their conformance to the Atlas Data Lake spec.
+
+Running these integration tests will require a running ``mongohoused``. Requirements
+for building ``mongohoused`` are as follows:
+
+  - Go version 1.13
+  - Building ``mqlrun``
+
+See the README file at ``https://github.com/10gen/mongohouse`` for more information.
+
+Building ``mongohoused`` locally
```

Comment:
The intent was to have a `driver-evergreen-tools` script that drivers could just use. All the build instructions are great, but simply doing this similar to atlas-proxy is by far a better setup. We do have artifacts that can be downloaded from S3 for both mongohoused and mqlrun. Building from master is fairly straightforward, which could be put into a driver-evergreen-tools script. Either way can be fine.

Reply:
I think relying on scripts to provision existing build artifacts is preferable to duplicating the build process for this project, so I'd like to see the original scope goal implemented. I'll leave it to @jstewart-mongo to sort that out with your team in the event he needs some pointers to the S3 sources.

Reply:
I have created a script for `driver-evergreen-tools` for this. The PR is located at https://github.com/jstewart-mongo/drivers-evergreen-tools/pull/1.

Reply:
I received an invite to your fork (not sure why it's private since https://github.com/mongodb-labs/drivers-evergreen-tools is already public), but I would have expected a PR against https://github.com/mongodb-labs/drivers-evergreen-tools directly. That way, any review stays on the common repository and it can be merged directly once approved (vs. presumably having to open a new PR after reviewing on your own fork).

Reply:
I'll resolve this as I opened a new comment thread on the revised README in this PR (and did suggest referencing the drivers-evergreen-tools repo).